zebrad/src/components/mempool/error.rs

```diff
@@ -22,4 +22,7 @@ pub enum MempoolError {
 
     #[error("transaction was not found in mempool")]
     NotInMempool,
+
+    #[error("transaction evicted from the mempool due to size restrictions")]
+    Victim,
```

Comment:
Maybe this wording is clearer and less emotionally loaded?
```suggestion
    Excess,
```